app/PiCalculator/pi.js

```diff
@@ -0,0 +1,222 @@
+var numOfPointsToTryPerClient = 100000000000;
+var batchSizePerUpdate = 5000000;
+
+PIData = new Mongo.Collection('pidata');
+var n = 5;
+
+var signedInWithFacebook = false;
+var signedInWithGoogle = false;
+var signedInWithLocalAccount = false;
+
+var initialized = false;
+
+var getCurrentTime = function () {
```

Comment:
It looks like you're trying to get the number of seconds since midnight (?) here.  It might be easier (and more universal) to use the number of seconds since the Unix epoch.  In JS, this is given in milliseconds (unlike in other languages which give the epoch time in seconds).  To get epoch time in seconds, you can do `Math.floor(Date.now() / 1000)`.  Or if you want more accuracy, just use `Date.now()` to get the time since the epoch in milliseconds.